### DISCUSSION TITLE
Confusión Instrucción Ejercicio 2 de Actividad 5

### DISCUSSION THREAD

**Discussion by gaborszabo17** [Taller-Abierto-de-Humanidades-Digitales/curso_python_extenso#45] [Dudas del curso]
Hola, Jairo,
no he podido hacer el segundo ejercicio porque no entiendo bien la siguiente instrucción:
_`Guarda la fecha de búsqueda en una variable y comprueba que su resultado sea correcto tanto para fechas anteriores como posteriores. Esto implica que debes crear una condición que valide si la consulta es para libros anteriores o posteriores a la fecha de búsqueda.`_
Exactamente, ¿qué significa comprobar el resultado correcto?
Dávid.

**Comment by jairomelo** [comment]
Hola @gaborszabo17 ,

Es decir, que haya una condición en la que el programa pueda decidir si, por ejemplo, la fecha de búsqueda es 1950, pueda saber si el listado resultante es para libros anteriores a la fecha de búsqueda o posteriores a la misma. Por ejemplo:

```
fecha_busqueda = 1950
anterior_posterior = anterior

## condicional ##

resultado = [libros anteriores a 1950]
```

Espero que sea un poco más claro. Me avisas :)

Saludos

**Reply by gaborszabo17** [reply]
Muchas gracias, Jairo. Intentaré aplicar el hint mañana antes de la clase.

**Reply by pipen77** [reply]
Tenía la misma duda, pero sobre todo sigo todavía sin entender si debemos usar obligatoriamente condicionales "or" o si esta bien si lo resolvemos con "if".

Saludos,

**Reply by jairomelo** [reply]
Hola @pipen77 ,

No es obligatorio el operador `or`. Todo depende de cómo puedes lograr que el programa tome la decisión de dar un listado u otro dependiendo de si son libros anteriores o posteriores a un fecha determinada.

**Reply by pipen77** [reply]
Genial, gracias Jairo.
